ui/raidboss/data/06-ew/raid/p4n.ts

```diff
@@ -0,0 +1,198 @@
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.AsphodelosTheFourthCircle,
+  triggers: [
+    {
+      id: 'P4N Elegant Evisceration',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A50', source: 'Hesperos' }),
+      response: Responses.tankCleave(),
+    },
+    // Strong proximity Aoe
+    {
+      id: 'P4N Levinstrike Pinax',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A3F', source: 'Hesperos', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Go to Corners',
+        },
+      },
+    },
+    // Knockback from middle
+    {
+      id: 'P4N Well Pinax',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A3E', source: 'Hesperos', capture: false }),
+      delaySeconds: 4,
+      response: Responses.knockback(),
+    },
+    {
+      id: 'P4N Acid Pinax',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A3C', source: 'Hesperos', capture: false }),
+      response: Responses.spread(),
+    },
+    {
+      id: 'P4N Lava Pinax',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A3D', source: 'Hesperos', capture: false }),
+      response: Responses.stackMarker(),
+    },
+    {
+      id: 'P4N Decollation',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A51', source: 'Hesperos', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P4N Bloodrake',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A40', source: 'Hesperos', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P4N Hell Skewer',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A4F', source: 'Hesperos', capture: false }),
+      response: Responses.awayFromFront(),
+    },
+    {
+      id: 'P4N Belone Coils',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '69DD', source: 'Hesperos', capture: false }),
+      delaySeconds: 2,
+      alarmText: (data, _matches, output) => {
+        if (data.role === 'tank')
+          return output.avoidTankTower!();
+        if (data.role === 'dps')
+          return output.avoidDpsTower!();
+        if (data.role === 'healer')
+          return output.avoidHealerTower!();
+      },
+      outputStrings: {
+        avoidTankTower: {
+          en: 'Avoid Tank Tower',
+        },
+        avoidDpsTower: {
+          en: 'Avoid DPS Towers',
+        },
+        avoidHealerTower: {
+          en: 'Avoid Healer Tower',
+        },
+      },
+    },
+    {
+      id: 'P4N Northerly Shift Slash',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A4A', source: 'Hesperos', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Go North, Stand on sides',
```

Comment:
What about this for brevity?

```suggestion
          en: 'Go North Edge',
```